amazonmusic.py

```diff
@@ -150,6 +151,23 @@ def _authenticate(self, r):
             'Accept-Language': 'en-US,en-GB;q=0.7,chrome://global/locale/intl.properties;q=0.3'
           },
           data = query)
+
+    if r.history and r.history[-1].is_redirect and AMAZON_MFA in r.history[-1].headers['Location']:
+      soup = BeautifulSoup(r.content, "html.parser")
+      query = {"otpCode": input("2FA Code:")}
```

Comment:
Given the library may be used through an app, or a voice interface, I think this will need to be pulled into an optional callout when constructing the `AmazonMusic` object.

Reply:
Yes i think so too.
A better solution is to provide a callback like this:

```Python
@amazonmusic.2fa
def 2fa(authorize):  # bad practice func name i know
    code = input('2FA Code: ')
    authorize(code)
```